src/Utilities/set_eval.jl

```diff
@@ -1,114 +1,121 @@
 
-Base.in(v, s::MOI.LessThan) = v <= s.upper
-Base.in(v, s::MOI.GreaterThan) = v >= s.lower
-Base.in(v, s::MOI.EqualTo) = v ≈ s.value
+function set_distance(v::Real, s)
```

Comment:
`set_distance` sounds like a setter.

Why not just `distance`, or (more explicit) `distance_to_set`?

Reply:
fixed in last commit